### DISCUSSION TITLE
Option to define an Hierarchy note with frontmatter rather than a note title

### DISCUSSION THREAD

**Discussion by bturloff** [SkepticMystic/breadcrumbs#491] [Ideas]
Instead of defining a hierarchy note with a specific note title i.e. "Hierarchy Note", can we define it with front matter i.e. "BC-hierarchy-note".

**Comment by michaelpporter** [comment]
Good news — this is exactly how v4 works now. There's no "Hierarchy Note" identified by its title anymore; **every** builder is driven by frontmatter (or properties), not by a magic note name.

- Plain parent/child: `up: "[[Parent]]"` / `down: "[[Child]]"` in frontmatter.
- The old "Hierarchy Note" is now a **List Note**, flagged with frontmatter: `BC-list-note-field: down` (the note's bullet list becomes the hierarchy).
- Same for Tag Notes (`BC-tag-note-field`), Dataview Notes (`BC-dataview-note-query`), Folder Notes, etc. — all frontmatter-configured.

So you define hierarchies entirely through frontmatter fields; no note needs a special title. Closing as resolved — covered by the v4 design. See the docs for the full list of builders.
